web/cypress/integration/consumer/create-consumer-with-proxy-mirror-form.spec.js

```diff
@@ -17,38 +17,52 @@
 /* eslint-disable no-undef */
 
 context('Create and delete consumer with proxy-mirror plugin form', () => {
-  beforeEach(() => {
-    cy.login();
-
-    cy.fixture('selector.json').as('domSelector');
-    cy.fixture('data.json').as('data');
-  });
-
   const selector = {
+    empty:'.ant-empty-normal',
+    username: '#username',
+    description: '#desc',
+    pluginCard: '.ant-card',
+    drawer: '.ant-drawer-content',
+    dropdown: '.rc-virtual-list',
+    disabledSwitcher: '#disable',
+    codeMirror: '.CodeMirror',
+    notification: '.ant-notification-notice-message',
+    notificationCloseIcon: '.ant-notification-close-icon',
     host: "#host",
     alert: "[role=alert]"
   }
```

Comment:
```suggestion
    notification: '.ant-notification-notice-message',
    notificationCloseIcon: '.ant-notification-close-icon',
     host: '#host',
     alert: '[role=alert]'
   }
```
It is best to use both single and double quotation marks

Reply:
![image](https://user-images.githubusercontent.com/2106987/117294079-ce10a900-aea4-11eb-86c4-6d730be2ac5d.png)

Why there have spaces?

Reply:
This is my problem. I didn't see the space gap clearly when commenting on my phone……

Reply:
ok